backend/layers/processing/dataset_metadata_update.py

```diff
@@ -137,68 +136,65 @@ def update_cxg(
 
     def update_metadata(
         self,
-        collection_version_id: CollectionVersionId,
-        dataset_version_id: DatasetVersionId,
+        old_dataset_version_id: DatasetVersionId,
```

Comment:
old, current, existing, we now have 3 ways of identifying nearly the same thing. Please take the time to consolidate the names.

Reply:
```suggestion
        current_dataset_version_id: DatasetVersionId,
```